iceoryx_hoofs/source/cxx/helplets.cpp

```diff
@@ -21,27 +21,33 @@ namespace iox
 {
 namespace cxx
 {
+// AXIVION DISABLE STYLE AutosarC++19_03-A5.2.4 : allow reinterpret_cast for the whole file
 void* alignedAlloc(const uint64_t alignment, const uint64_t size) noexcept
 {
     // -1 == since the max alignment addition is alignment - 1 otherwise the
     // memory is already aligned and we have to do nothing
     // low-level memory management
     // NOLINTNEXTLINE(cppcoreguidelines-owning-memory, cppcoreguidelines-no-malloc, hicpp-use-auto)
-    uint64_t memory = reinterpret_cast<uint64_t>(malloc(size + alignment + sizeof(void*) - 1));
+    // AXIVION Next Line AutosarC++19_03-M5.2.9 : Pointer conversion is safe here.
```

Comment:
Is this justification sufficient or should we state here why the pointer conversion is safe?

Reply:
@FerdinandSpitzschnueffler sure shall re frame the sentence may be like "Cast is required since we perform low level byte manipulation to access stored meta-data."